micronutrientsupport/micronutrient-support-tool
# More use of PATCH endpoint for interventions

- startup costs with dialogs patching data changes
- recurring costs with dialogs patching data changes
- updated industry information to track percentage data not just number
- added blue style to inputs which are modified

## How to test
- look in the interventions pages at recurring costs and startup costs. Open the table dialogs, change values (which should show as blue when edited), confirm to close and save the changes, then reopen the dialog/refresh page and data should persist

## Note
- `.toFixed(2)` on input values are not being honoured. The appDecimal pipe ensures that user inputs are 2dp but if the original data is more than 2dp the `toFixed(2)` does not work.
